extensions/panache/hibernate-orm-panache/deployment/src/main/java/io/quarkus/hibernate/orm/panache/deployment/EntityField.java

```diff
@@ -1,5 +1,7 @@
 package io.quarkus.hibernate.orm.panache.deployment;
 
+import io.quarkus.panache.common.deployment.JavaBeanUtil;
```

Comment:
new import but not used?

Reply:
It is used, it's a result of moving this type to a different package, so it did not use to be imported.